src/registrar/models/domain.py

```diff
@@ -336,14 +336,11 @@ def statuses(self) -> list[str]:
 
         A domain's status indicates various properties. See Domain.Status.
         """
-        # implementation note: the Status object from EPP stores the string in
-        # a dataclass property `state`, not to be confused with the `state` field here
-        if "statuses" not in self._cache:
-            self._fetch_cache()
-        if "statuses" not in self._cache:
-            raise Exception("Can't retreive status from domain info")
-        else:
-            return self._cache["statuses"]
+        try:
+            return self._get_property("statuses")
+        except KeyError:
+            logger.error("Can't retrieve status from domain info")
+            return []
```

Comment:
I suggest leaving it penciled in and commenting it out so that future devs understand that:

"""
        We will not implement this. Statuses are set by the registry
        when we run delete and client hold, and these are the only statuses
        we will be triggering.
        """